### DISCUSSION TITLE
Feature Req: Integrate postlight parser into epub creation / reader mode

### DISCUSSION THREAD

**Discussion by foundObjects** [plateaukao/einkbro#223] [Ideas]
I've noticed that epub creation doesn't always do a great job of stripping content down to bare text.

A good repro case here is Clarkesworld magazine, open any of the short stories, save one as an epub and then open the resulting file in an epub reader (I'm using KOReader.) Jump to the end of the document and you'll find the text embedded in many nested tables, about one per page.

Many of my longer saved pages from other sites end up with similar problems. Preparsing each page with postlight would eliminate any need to roll a reader mode parser (or maintain it) and improve the quality of content in reader view and saved epubs.

**Comment by plateaukao** [comment]
Thank you for the advice.
It seems that postlight only has javascript implementation? If so, it's hard to include it in EinkBro.
pull requests for this feature are always welcomed. :)
